themes/default/content/blog/zephyr-code-placement-and-stacks/index.md

```diff
@@ -0,0 +1,140 @@
+---
+title: "Zephyr: Code Placement and Stacks"
+
+# The date represents the post's publish date, and by default corresponds with
+# the date this file was generated. Posts with future dates are visible in development,
+# but excluded from production builds. Use the time and timezone-offset portions of
+# of this value to schedule posts for publishing later.
+date: 2023-02-15T13:01:26-07:00
+
+# Use the meta_desc property to provide a brief summary (one or two sentences)
+# of the content of the post, which is useful for targeting search results or social-media
+# previews. This field is required or the build will fail the linter test.
+# Max length is 160 characters.
+meta_desc: This is the first in a series of blog posts on recommended practices for using Pulumi. In this first post, we discuss code placement and Pulumi stacks.
+
+# The meta_image appears in social-media previews and on the blog home page.
+# A placeholder image representing the recommended format, dimensions and aspect
+# ratio has been provided for you.
+meta_image: meta.png
+
+# At least one author is required. The values in this list correspond with the `id`
+# properties of the team member files at /data/team/team. Create a file for yourself
+# if you don't already have one.
+authors:
+    - scott-lowe
+    - christian-nunciato
+    - aaron-kao
+
+# At least one tag is required. Lowercase, hyphen-delimited is recommended.
+tags:
+    - development-environment
+    - cloud-engineering
+
+# See the blogging docs at https://github.com/pulumi/pulumi-hugo/blob/master/BLOGGING.md.
+# for additional details, and please remove these comments before submitting for review.
+---
+
+This is the first in a series of blog posts where we'll explore how a fictional company---Zephyr Archaeotech Emporium---uses Pulumi to manage their online retail store. In this first post, we'll explore a couple common questions that users ask when working with Pulumi; specifically, where should I store my Pulumi code? And how do I support multiple environments with Pulumi? This post will provide some guidance and recommendations around these topics, using Zephyr and their online store as the use case.
+
+<!--more-->
+
+Here are links to all the blog posts in the series: <!--The names of these posts and the order in which we publish them is mutable-->
+
+**Zephyr: Code Placement and Stacks** (this post)
+
+Zephyr: Structuring Their Pulumi Projects
```

Comment:
should we call out that these are coming in the future? since it says "here are the links to all the blog posts in the series", the reader might assume they are existing and think links are just not working.

Reply:
I've added some text to indicate that if an entry isn't linked it hasn't been published yet

Addressed in 35635a95